starter-code/src/components/footer.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "../styles/footer.css";
+
+export default function footer() {
+    return (
+        <footer className="footer">
+            <div className="logos">
+                <img src="./images/icon1.png" alt="" />
+                <h2>Declarative</h2>
+                <p>React makes it<br />painless to create<br />interactive UIs.</p>
+            </div>
```

Comment:
You are creating 4 times the same "template".
You can break this down into a component : )